### ISSUE TITLE
Documenting how to make Medley's window or text bigger

### ISSUE THREAD

**Issue by pamoroso** [Interlisp/medley#1622]
Some new users with large screens ask how to make the window or text of Medley bigger, see for example [here](https://fosstodon.org/@jdboyd@treehouse.systems/112008401110643894) and [here](https://fosstodon.org/@dziban@functional.cafe/112172917718445115).

Since this may be a FAQ it would help to document how to do this such as the `--geometry` and `--screensize` options of `medley` and the `-pixelscale` option of Maiko.

**Comment by MarceColl** [comment]
It would also be nice to tell about the draw buffer limitation, since there is an upper limit on how big the screen can be due to limitations in the VM.

**Comment by khinsen** [comment]
Yes, please provide some documentation for common cases!

I tried to run Medley on my 13" Dell laptop running Debian 12. Everything works fine as advertised, but I need a magnifying lens to read any text on the screen.

I found many discussions about this problem, but no obvious solution. Some people say I should recompile with SDL instead of X, so that I can use the `-ps` option, but there are no instructions. Others say I should use `xrandr` but provide no instructions nor a working example. There are hints that using VNC would allow scaling, and VNC actually works, contrary to what the installation instructions say, but I have no idea how to profit from VNC for scaling.

**Comment by pamoroso** [comment]
In [this post](https://journal.paoloamoroso.com/how-to-build-medley-interlisp) I explained how to build Medley from source with instructions for X11 and SDL. Once you build for SDL try this command:

```
medley -a -e -n -ps 2
```

The command launches plain Medley (i.e. no VNC) and doubles the pixel size with `-ps 2`. Depending on your screen you can try other scaling factors such as 3 or 4.

I use Medley with VNC but I have no idea whether TigerVNC supports scaling, and how. I don't run TigerVNC directly as I use the `medley` script to drive the VNC server and viewer with the default options.

**Comment by khinsen** [comment]
Thanks, I will try those instructions!

**Comment by pamoroso** [comment]
@khinsen Did the instructions work for you?

**Comment by khinsen** [comment]
@pamoroso No, the scaling factor had no effect. That's why I started working on compiling Medley myself, via Guix, in order to have a better understanding of how it works and more buttons I can turn for experimenting. Then summer vacations happened... I need to get back to this!

**Comment by pamoroso** [comment]
@khinsen Any updates on whether the instructions work?

**Comment by khinsen** [comment]
No, sorry. Too much to do for 24 hour days!

**Comment by pamoroso** [comment]
No problem. Since we [documented how to make Medley's window bigger](https://interlisp.org/project/faqs/#i-have-a-large-display--medley-is-so-tiny-its-hard-to-read) I'm going to close this issue for the time being.
